Assets/MRTK/Providers/Oculus/XRSDK/OculusXRSDKDeviceManager.cs

```diff
@@ -275,20 +275,29 @@ private OculusHand GetOrAddHand(Handedness handedness, OVRHand ovrHand)
             var inputSource = inputSystem?.RequestNewGenericInputSource($"Oculus Quest {handedness} Hand", pointers, inputSourceType);
 
 
-            OculusHand handController = new OculusHand(TrackingState.Tracked, handedness, inputSource);
-            handController.InitializeHand(ovrHand, MRTKOculusConfig.Instance.CustomHandMaterial);
+            OculusHand handDevice = new OculusHand(TrackingState.Tracked, handedness, inputSource);
+            handDevice.InitializeHand(ovrHand, MRTKOculusConfig.Instance.CustomHandMaterial);
```

Comment:
I know this is already-merged code, but this `MRTKOculusConfig.Instance` pattern goes against a lot of the established MRTK patterns. This feels like a lot of it should be part of a data provider profile, with accessors on the data provider instead of on some singleton ScriptableObject.

Reply:
Would also help with an assigned teleport action (referenced at https://github.com/microsoft/MixedRealityToolkit-Unity/pull/8386#discussion_r480512635)